scripts/aznfs_install.sh

```diff
@@ -75,6 +115,111 @@ use_dnf_or_yum()
     fi
 }
 
+is_new_version_available()
+{
+    local current_version_integer=$(echo $1 | tr -d ".")
+    local latest_version_integer=$(echo $2 | tr -d ".")
+
+    [ $latest_version_integer -gt $current_version_integer ]
+}
+
+# Function to perform AZNFS update.
+perform_aznfs_update() 
+{
+    if [ "$install_cmd" == "apt" ]; then
+        AZNFS_RELEASE="aznfs-${RELEASE_NUMBER}-1"
+        package_name=${AZNFS_RELEASE}_amd64.deb
+    elif [ "$install_cmd" == "zypper" ]; then
+        AZNFS_RELEASE_SUSE="aznfs_sles-${RELEASE_NUMBER}-1"
+        package_name=${AZNFS_RELEASE_SUSE}.x86_64.rpm
+    else
+        AZNFS_RELEASE="aznfs-${RELEASE_NUMBER}-1"
+        package_name=${AZNFS_RELEASE}.x86_64.rpm
+    fi
+
+    # Use wget to download the package, and check for success.
+    wget_output=$(wget --timeout=120 "https://github.com/Azure/AZNFS-mount/releases/download/${RELEASE_NUMBER}/${package_name}" -P /tmp 2>&1)
+    if [ $? -ne 0 ]; then
+        eecho "Failed to download the package using wget, exiting!"
+        eecho "$wget_output"
+        exit 1
+    fi
+    
+    if [ ! -f "/tmp/${package_name}" ]; then
+        eecho "[BUG] Downloaded package file '/tmp/${package_name}' not found, installation aborted!"
+        exit 1
+    fi
+    
+    # Check if the downloaded file exists before proceeding with installation.
+    if [ "$install_cmd" == "zypper" ]; then
+        install_output=$($install_cmd install --allow-unsigned-rpm -y "/tmp/${package_name}" 2>&1)
+    else
+        install_output=$($install_cmd install -y "/tmp/${package_name}" 2>&1)
+    fi
+    install_error=$?
+    rm -f "/tmp/${package_name}"
+
+    if [ $install_error -ne 0 ]; then
+        eecho "[FATAL] Error installing AZNFS $RELEASE_NUMBER (Error: $install_error). See '$install_cmd' command logs for more information"
```

Comment:
Since we have used "AZNFS version $RELEASE_NUMBER" everywhere else, would be good to keep it consistent.